### ISSUE TITLE
releasewarrior command line should link to how to documents

### ISSUE THREAD

**Issue by mozbhearsum** [mozilla-releng/releasewarrior-2.0#54]
"release status" seems like the primary way in which people interact with releasewarrior for inflight releases. It would be helpful to link to howto documents for pending tasks, rather than requiring people to go to the .md pages on github to find them.

**Comment by srfraser** [comment]
I agree, although the urls are a bit long. We could make a call out to a url shortener (on creation, maybe?).  mzl.la uses bit.ly under the hood. Any objection to doing the same?

**Comment by srfraser** [comment]
See also #11
